global_costmap:
  global_frame: map
  robot_base_frame: $(arg robot)/base_link

  update_frequency: 0.1
  publish_frequency: 0.1
  transform_tolerance: 0.5

  static_map: true
 
